### ISSUE TITLE
One email with no sender leads to endless errors

### ISSUE THREAD

**Issue by justpusher** [osTicket/osTicket#3504]
I'm using osticket 1.9.12 configured to fetch email messages via IMAP and moving fetched messages to a folder. 
Recently a mailbox received a message with no From address. This message could not be imported as a ticket:
`        if(!($headerinfo=imap_headerinfo($this->mbox, $mid)) || !$headerinfo->from)    
            return null;`
So, this message is being skipped and left on the mailbox. Forever. 
If there were no other messages on the cron run, the script counts 0 imported tickets and 1 error. 
It is considered as "Excessive errors":
`if($errors>$msgs) {    
            $warn=sprintf(_S('Excessive errors processing emails for %1$s/%2$s. Please manually check the inbox.'),    
                    $this->getHost(), $this->getUsername());`

This error occurs on almost every cron run, causing the endless errors being sent to the system email.

I think you should change the way you detect excessive errors. Or, maybe you should not consider the skipped message as an error. 

P.S. I've checked the latest version 1.9.15, and this code has not been changed.

**Comment by protich** [comment]
I think this is a legit error that warrants an admin looking into it. I believe the system alerts when 80% of per fetch count fails -- in your case it's 100%.

**Comment by ntozier** [comment]
I agree that this is legit error and that it should trigger an error because it requires manual admin intervention to rectify, which of course means we need to see the error.

**Comment by 9joshua** [comment]
The error should be more specific. I have had this problem for a while and didn't know what caused it until I saw this issue thread. i.e....
```
One or more emails in the inbox for <inbox_email_address> do not contain a FROM address. Please manually check the inbox and remove the email(s)"
```
The current error message `Please manually check the inbox.` is ambiguous because like me, other people might think it simply means you need to manually process emails, not check for email header errors. I assumed the issue was an intermittent problem with my server's ability to retrieve email.
